integrationtests/base.go

```diff
@@ -0,0 +1,139 @@
+package integrationtests
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"net"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+)
+
+// UpWg used for setting up the wireguard interface
+func UpWg(wgConfig, wgConfDir string, t *testing.T) (string, error) {
+	tempdir := t.TempDir()
+	f, err := os.Create(tempdir + "/test.conf")
```

Comment:
use `Join` function instead of hardcoding the path separator https://pkg.go.dev/path/filepath#Join